src/main/webapp/scripts/map.js

```diff
@@ -12,82 +12,145 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Global Icon Variables */
+var yellowIcon;
+var blueIcon;
+var purpleIcon;
+var redIcon;
+var greyIcon;
 
+/* Hard-coded Recommendations */
+const recommendations = [
```

Comment:
It's more convenient and readable to have this as an array of [objects](https://www.w3schools.com/js/js_objects.asp) instead of a 2D array. So when you read the parameters of a recommendation below it'll be for example `recommendation.name` instead of `recommendation[1]`

Reply:
Great suggestion, thanks!

Reply:
Great suggestion, thanks!